notebooks/notebooks/iclr2022/estimate-gpu-days-adv.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    gpu_type_to_seconds = {}
</pre><p>Could also use a defaultdict(int) to simplify code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>